Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageBlocksStateManager.swift

```diff
@@ -291,12 +291,19 @@ final class EditorPageBlocksStateManager: EditorPageBlocksStateManagerProtocol {
                     guard let self, let id = targetDocument.children.last?.id,
                           let details = targetDocument.details else { return }
                     if !details.isList {
-                        try await move(position: .bottom, targetId: targetDocument.objectId, dropTargetId: id, blocksIds: filteredBlocksIds)
-                        toastPresenter.showObjectCompositeAlert(
```

Comment:
why `try await` syntaxis doesn't work? `toastPresenter` was called after move. `completion` called at the same time 🤔

Reply:
it works, but there are some condition inside `move` that can go to return after check and we don't need to call `completion`

Reply:
Maybe add result reason, to use swift concurrency sintaxis? I expect the completion to be call every time

Reply:
or add specific error

Reply:
I will add result 🙌